src/mozmlops/templates/README.md

```diff
@@ -52,6 +52,22 @@ $METAFLOW_PROFILE=local python template_flow.py run --offline True
 
 Eventually you should see the flow finish with "Task finished successfully." Once you see that, you know you've got a working minimal flow: you're now ready to start putting your model training code _into_ this flow to run it remotely. Comments _inside_ the template file should help you understand where to put that code, and demonstrate some of the tools Metaflow makes available to you.
 
+ ## Using Docker with Model Orchestration Flows
+
+Flows in Metaflow permit you to include dependencies in each of your steps in two different ways:
+
+1. Adding the `@pypi` or `@conda` decorator and installing dependencies as each step runs ([documentation here](https://docs.metaflow.org/scaling/dependencies/libraries))
+2. Using a custom docker image with your dependencies in it, as described in [this documentation](https://docs.metaflow.org/scaling/dependencies/containers)
+   3. An important note: the above linked documentation says you can use docker images and the pypi and conda decorators together. According to the maintainers of the Outerbounds product that we use at Mozilla, this does not function as documented; instead, both the pypi and conda decorators start from a fresh environment. So if you specify _both_, the flow will _run_, but it will _fail_ if you try to import something in the step that exclusively lives in the docker image. As of August 2024, the Mozilla MLOps team has confirmed this behavior.
```

Comment:
I _do_ mean for this to be a subpoint of the second point, rather than a third point, as it is not one of the ways you include dependencies in your flow but rather a detail you should know if you're using the second way. However, I agree with you that this wording can be clarified!